CRM/Civicase/Hook/BuildForm/ButtonsLableChange.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Changes the label of buttons in the create pdf popup.
+ */
+class CRM_Civicase_Hook_BuildForm_ButtonsLableChange {
```

Comment:
Just thought that we should rename this class to something more specific. `ButtonsLableChange` is too generic. Can we try `PdfFormButtonsLabelChange`